src/components/Apps/AccountListTable/const.js

```diff
@@ -7,11 +7,23 @@ export const connectivityMeta = {
     faChoices: {
       '-': '',
       ok: 'fa-check-circle',
-      err: 'fa-times-circle'
+      err: 'fa-times-circle',
+      auth_err: 'fa-times-circle',
+      sudo_err: 'fa-times-circle',
+      password_err: 'fa-times-circle',
+      openssh_key_err: 'fa-times-circle',
+      ntlm_err: 'fa-times-circle',
+      create_dir_err: 'fa-times-circle'
     },
     classChoices: {
       ok: 'text-primary',
-      err: 'text-danger'
+      err: 'text-danger',
+      auth_err: 'text-danger',
+      sudo_err: 'text-danger',
+      password_err: 'text-danger',
+      openssh_key_err: 'text-danger',
+      ntlm_err: 'text-danger',
+      create_dir_err: 'text-danger'
     },
     getText({ cellValue }) {
       if (cellValue?.value === '-' || cellValue?.value === 'unknown') {
```

Comment:
In the given code, changes can be made to improve readability.

Changes:

1. Ensure there is a space after `const` keyword.
2. Add proper line endings (Unix style).
3. Use double quotes instead of single quotes around keys in object property names.

Example Improved Code:
    export const connectivityMeta = {
       authErr: 
         {
           '-': '',  
          // Remove this comment if all error messages are strings,
        };  

The improved code has better formatting, making it easier to read and navigate through.